pallets/gear-program/README.md

```diff
@@ -0,0 +1,11 @@
+# Tracking and charging fee for on-chain resources usage
```

Comment:
This should probably be replaced with the information relevant to the pallet in question.